molgenis-dataexplorer/src/main/resources/js/dataexplorer-diseasematcher.js

```diff
@@ -622,18 +622,19 @@
 			}
 			
 			for (var propt in clinicalSynopsis){
-				// remove links, id's between { and }
-				var phenotype = clinicalSynopsis[propt].replace(/ *\{[^}]*\} */g, '');
+				// remove links, id's between { and }, and linebreaks
+				var phenotypes = clinicalSynopsis[propt].replace(/ *\{[^}]*\} */g, '');
+				phenotypes = phenotypes.split(";");
 				
-				// give each phenotype in one string it's own line
-				phenotype = replaceAll(phenotype, ';', '');
-				phenotype = phenotype.replace(lineBreaks, '<br />');
-				
-				if (propt == 'Inheritance' || propt == 'inheritance'){
-					synopsisParagraph.prepend('<span class="diseasematcher label label-success">' + phenotype + '</span><br />');		
-				}else{							
-					synopsisParagraph.append(phenotype + '<br />');
-				}
+				for (var phen in phenotypes){
+					if (propt == 'Inheritance' || propt == 'inheritance'){
+						synopsisParagraph.prepend('<span class="diseasematcher label label-success">' + phenotypes[phen] + '</span><br />');		
```

Comment:
escape span content to prevent XSS:
a) use Handlebars template
b) use JQuery .html('unescaped text') on span element